source/documentation/deploying_apps/index.erb

```diff
@@ -74,23 +74,27 @@ A possible exception to this is if your org is mature and has pre-existing space
 * Instances will be restarted if they [exceed memory limits](managing_apps.html#quotas).
 * Your application should write all its log messages to `STDOUT`/`STDERR`, rather than a log file.
 
-
 ## Deploying private apps
 
-You can deploy apps so they are not publicly accessible to everyone through the internet.
+By default, all apps you deploy on Cloud Foundry are [publicly accessible](deploying_apps.html#deploying-public-apps) to everyone through the internet.
+
+If you need to restrict access to a public app, for example to whitelist IPs, you should refer to the documentation on [route services](/deploying_services/route_services/#route-services).
+
+If you do not want your apps to be publicly accessible at all, you must deploy your apps on the `apps.internal` domain. Two common use cases for this are that you want to:
+
+- deploy a task scheduling app on the GOV.UK PaaS that does not need to be publicly accessible
+- test a beta version of an app in your production environment that only your team can access
```

Comment:
Private apps aren't accessible from the outside, without respect for who the user is. In this case, the thing that can access the beta version is other apps with configured routes to it.

I think in hindsight, there's probably a better example than this. I'll ask in Slack.

Reply:
@jonathanglassman From Friday's discussion, it looks like the most common usage of private apps is for deploying micro-service (the technical term) architected services (the citizen-facing term).

Reply:
@AP-Hunt thanks. So what are architected services?